src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -182,7 +188,13 @@ void SensorFusion::updateWorld(
             }
         }
         friendly_team.setUnavailableRobotCapabilities(robot_id, unavailableCapabilities);
+
+        if (!estop_enabled_in_msg && robot_status_msg.estop_enabled())
+        {
+            estop_enabled_in_msg = true;
+        }
     }
+    estop_enabled = estop_enabled_in_msg;
```

Comment:
the temporary `estop_enabled_in_msg` starts off false and we stop receiving robot status msgs, we unconditionally disable the e-stop here

would it be better to just set the `estop_enabled` directly in the loop? that way we can "latch" to the e-stop enabled state if we ever see a real robot

Reply:
The state of `estop_enabled` is preserved in the `wifi_backend`, and Once estop is set it cannot be unset. So not receiving any more robot status msgs wouldnt disable it. 

I was originally thinking of the latch but decided against it because i think it would be the only member of `sensor_fusion` to behave that way. However i dont mind doing it that way